stac/disasters/hurricane-harvey/hurricane-harvey-0831/20170831_172754_101c/20170831_172754_101c.json

```diff
@@ -19,7 +20,6 @@
     "constellation": "planetscope",
     "created": "2017-10-27T12:14:07Z",
     "eo:cloud_cover": 2,
-    "pl:anomalous_pixels": 0.01,
     "pl:ground_control": true,
     "pl:item_type": "PSScene4Band",
```

Comment:
It's an old PSScene4Band, should we leave this as it is? For PSScene the pl:clear_percent is missing, but it's also not available in https://api.planet.com/data/v1/item-types/PSScene/items/20170831_172754_101c

Reply:
Huh, this seems like we didn't fully process the historical data values. I think anomalous pixels was from our first UDM, and then when we did UDM2 we generated all the clear pixel percents and the like. This looks like it has UDM2, but didn't get the updated values. I think it's fine to just take the anomalous pixels off.